Backend/app/rider-platform/rider-app/Main/src/SharedLogic/MessageBuilder.hs

```diff
@@ -224,6 +225,31 @@ buildFRFSTicketBookedMessage pOrgId req = do
           & T.replace (templateText "TICKET_PLURAL") ticketPlural
           & T.replace (templateText "URL") url
 
+buildFRFSTicketCancelMessage :: (EsqDBFlow m r, CacheFlow m r, HasFlowEnv m r '["urlShortnerConfig" ::: UrlShortner.UrlShortnerConfig]) => Id DMOC.MerchantOperatingCity -> Id DPO.PartnerOrganization -> BuildFRFSTicketBookedMessageReq -> m (Text)
+buildFRFSTicketCancelMessage merchantOperatingCityId pOrgId req = do
+  merchantMessage <-
+    QMM.findByMerchantOperatingCityIdAndMessageKey merchantOperatingCityId DMM.METRO_TICKET_BOOKING_CANCELLED
```

Comment:
Add the sql migration for inserting the template as well.

Reply:
Sure , once the template is confirmed i will add the migarations